packages/migration/src/migrations/hearth/20240923141025-change-task-input-output-informant-phone-value-id-value-code.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+
+import { Db, MongoClient } from 'mongodb'
+
+const LEGACY_INFORMANT_PHONE_VALUE_ID =
```

Comment:
Read this carefully, let me know if this is actually a custom pattern.

Reply:
<img width="729" alt="Screenshot 2024-09-23 at 14 48 37" src="https://github.com/user-attachments/assets/a0fed810-755b-4fbf-a539-9a2f1eba1edb">

corresponds to this